### PR TITLE
Kyma migration to OWNERS from CODEOWNERS

### PR DESCRIPTION
**Pull request by Ressetkk**
This Pull Request contains changes related to migration from GitHub CODEOWNERS to OWNERS files. Everyone should be able to push their changes to this PR.

## Introduction
THe OWNERS file contain one structure from the two available. The accepted names are GitHub usernames, or aliases from OWNERS_ALIASES file.
```yaml
reviewers:
  - owner
  - owner-from-owners-aliases
approvers:
  - approver
  - approver-from-owners-aliases
labels:
  - auto-added-label-to-prs
```
or
```yaml
filters:
  '.*':
    reviewers:
      - ...
      - ...
    approvers:
      - ...
    labels:
      - ...
  'regex': # ONLY LABELS WORK WITH REGEX FILTER
    labels:
      - ...
```

OWNERS_ALIASES file contains aliases for GitHub usernames, which can be used in OWNERS files, instead of the regular usernames.
```yaml
aliases:
  alias1:
    - ...
    - ...
  alias2:
    - ...
    - ...
```

## How to update OWNERS files
Push your changes to the PR branch, copy-paste and fill-up the following checklist in the comment after you push the changes:
```
**Area group name**:
**Directories updated:**
* ...

**Required checks**
* [ ] File ownership documented in internal team documentation
* [ ] OWNERS_ALIASES are properly documented
* [ ] OWNERS files reflect current state compared to current CODEOWNERS
* [ ] Commits are squashed
```

If the team wishes to use OWNERS_ALIASES make sure to also add or update the desired change in the OWNERS_ALIASES file, and reflect the names in your OWNERS file. Please refer to initial configuration of kyma components in the PR.

If you have any questions please write them down. I'll try to answer all questions ASAP.

/kind deprecation

### CODE REVIEW

**Comment by Disper** [comment]
It could be worth adding that @Ressetkk, in his initial commit provided some examples how it could be done for the `components` directory. The remaining directories still needs to have the `OWNERS` and `OWNERS_ALIASES` configured.
Additionally, even the initially provided `OWNERS` files can adjusted to the code owners needs, including splitting/merging proposed aliases or having having different `OWNERS` configuration deeper in your directory structures.

File: OWNERS_ALIASES
Comment (majakurcius) [inline OWNERS_ALIASES:16]:
```suggestion
    - NHingerl
```
@alexandra-simeonova has moved to another unit with the Luigi team, additionally, @NHingerl is missing from the list

Comment (Ressetkk) [reply OWNERS_ALIASES:16]: @majakurcius please make commit that reflect changes for technical-writers, as per instruction in PR body.

**Comment by majakurcius** [comment]
**Area group name**:
- documentation-reviewers:
- documentation-approvers:
- technical-writers:
**Directories updated:**
- OWNER-ALIASES > aliases

**Required checks**
* [x] File ownership documented in internal team documentation
* [x] OWNERS_ALIASES are properly documented
* [x] OWNERS files reflect current state compared to current CODEOWNERS
* [x] Commits are squashed

**Comment by Ressetkk** [comment]
@k15r just add needed changes to this PR.

**Comment by dennis-ge** [comment]
**Area group name**: Observability
**Directories updated:**
* components/telemetry-operator
* resources/logging
* resources/monitoring
* resources/kiali
* resources/tracing
* resources/telemetry
* resources/istio-configuration/templates/monitoring

**Required checks**
* [] File ownership documented in internal team documentation
* [x] OWNERS_ALIASES are properly documented
* [x] OWNERS files reflect current state compared to current CODEOWNERS
* [x] Commits are squashed

**Comment by mvshao** [comment]
**Area group name**: application-connector
**Directories updated:**
- resources/application-connector
- resources/compass-runtime-agent
- tests/application-connector-test
- tests/application-gateway-legacy-tests
- tests/application-gateway-tests
- tests/application-operator-tests
- tests/application-registry-tests
- tests/compass-runtime-agent
- tests/connection-token-handler-tests
- tests/connector-service-tests

**Required checks**
* [x] File ownership documented in internal team documentation
* [x] OWNERS_ALIASES are properly documented
* [x] OWNERS files reflect current state compared to current CODEOWNERS
* [x] Commits are squashed

**Review by franpog859 — DISMISSED** [review]

**Review by Sawthis — DISMISSED** [review]

**Comment by klaudiagrz** [comment]
@Ressetkk Please note that the area name changed from `service-catalog` to `service-management`.

**Area group name**: service-management
**Directories updated:**
* OWNERS_ALIASES
* components/application-broker
* components/binding
* components/service-binding-usage-controller
* components/uaa-activator
* tests/fast-integration/kcp
* tests/fast-integration/kyma-environment-broker
* tests/fast-integration/skr-kyma-to-kyma2-upgrade
* tests/fast-integration/skr-nightly
* tests/fast-integration/skr-svcat-migration-test
* tests/fast-integration/skr-test
* tests/service-catalog

**Required checks**
* [ ] File ownership documented in internal team documentation
* [ ] OWNERS_ALIASES are properly documented
* [ ] OWNERS files reflect current state compared to current CODEOWNERS
* [ ] Commits are squashed

**Comment by Ressetkk** [comment]
> What is the difference between approvers and reviewers? @Ressetkk

Reviewers are people who will be picked up by the Prow to do a review. Approvers are people who can approve the PR once the review process is finished and the change is approved for merge - basically those are the actual codeowners. Approvers can be reviewers, but the best practice is to keep those two lists different from each other. Approver's approving review counts as completed review.

**Review by jeremyharisch — DISMISSED** [review]

**Comment by dennis-ge** [comment]
**Area group name**: observability
**Directories updated:**
* OWNERS_ALIASES
* tests/perf

**Required checks**
* [x] File ownership documented in internal team documentation
* [x] OWNERS_ALIASES are properly documented
* [x] OWNERS files reflect current state compared to current CODEOWNERS
* [ ] Commits are squashed

**Comment by mjakobczyk** [comment]
**Area group name**: api-gateway
**Directories updated:**
* OWNERS_ALIASES
* resources/api-gateway/OWNERS
* resources/ory/OWNERS
* tests/integration/api-gateway/OWNERS

**Area group name**: service-mesh
**Directories updated:**
* OWNERS_ALIASES
* resources/istio/OWNERS
* resources/istio-resources/OWNERS
* resources/istio-configuration/OWNERS

**Other directories updates:** (not coved by aliases)
* resources/certificates/OWNERS
* resources/cluster-users/OWNERS

**Required checks**
* [x] File ownership documented in internal team documentation
* [x] OWNERS_ALIASES are properly documented
* [x] OWNERS files reflect current state compared to current CODEOWNERS
* [x] Commits are squashed

**Review by pPrecel — DISMISSED** [review]

**Review by Disper — APPROVED** [review]

**Review by Sawthis — APPROVED** [review]

**Review by pPrecel — APPROVED** [review]

**Review by Sawthis — APPROVED** [review]

**Pull request merged**
